charts/node-analyzer/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: node-analyzer
+description: Sysdig Node Analyzer
+
+# currently matching Sysdig's appVersion 1.12.59
+version: 1.0.0
+appVersion: 12.3.1
```

Comment:
Not entirely sure if appVersion should be 12.3.1? It was copied from sysdig chart but it doesn't have anything to do with the node analyzer. Is there a different version value that makes more sense?